ebsco_adapter/terraform/indexer_lambda.tf

```diff
@@ -55,15 +69,65 @@ resource "aws_iam_role_policy" "indexer_lambda_policy" {
   policy = data.aws_iam_policy_document.read_ebsco_adapter_bucket.json
 }
 
-resource "aws_sns_topic_subscription" "indexer_lambda_subscription" {
+resource "aws_iam_role_policy" "indexer_lambda_sqs_policy" {
+  role   = module.indexer_lambda.lambda_role.name
+  policy = data.aws_iam_policy_document.allow_sqs_receive_message.json
+}
+
+# Add an SQS queue which will collect messages from SNS
+resource "aws_sqs_queue" "indexer_message_queue" {
```

Comment:
You may want to use https://github.com/wellcomecollection/terraform-aws-sqs, which includes some platform standard things that could be useful (alerting / DLQ).

Reply:
I assume I should use the `platform_dlq_non_empty_alarm` SNS topic for the DLQ notifications?

https://eu-west-1.console.aws.amazon.com/sns/v3/home?region=eu-west-1#/topic/arn:aws:sns:eu-west-1:760097843905:platform_dlq_non_empty_alarm

Reply:
That seems right to me ... this will result in slack alerts if we have items on the DLQ.